config/zones.json

```diff
@@ -1,725 +1,525 @@
 {
```

Comment:
Can you sort keys alphabetically? The changes will be much smaller in that case :)